tests/test_cnn.py

```diff
@@ -0,0 +1,43 @@
+from wellcomeml.ml import CNNClassifier, KerasVectorizer
+from sklearn.pipeline import Pipeline
+import numpy as np
+
+
+def test_vanilla():
+    X = [
+        "One",
+        "One only",
+        "Two nothing else",
+        "Two and three"
+    ]
+    Y = np.array([0,0,1,1])
+
+    model = Pipeline([
+        ('vec', KerasVectorizer()),
+        ('clf', CNNClassifier())
+    ])
+    model.fit(X, Y)
+    assert model.score(X, Y) > 0.6
```

Comment:
Previously we had 0.4 for Bert classifier, and used to fail. Isn't this going to fail sometimes too?

Reply:
I realised that i need to beat luck. A random guess would perform 0.5.

Reply:
If we have problems we can fallback to ensuring loss is decreased